client/src/GlobalStyle.js

```diff
@@ -64,4 +65,16 @@ export default createGlobalStyle`
         height: auto;
         display: block;
     }
+
+    .rmsc {
+        --rmsc-main: var(--primary);
+        --rmsc-hover: #f1f3f5;
+        --rmsc-selected: var(--secondary-light);
+        --rmsc-border: none;
+        --rmsc-gray: var(--primary-darkest);
+        --rmsc-bg: var(--grey-lightest);
+        --rmsc-p: 10px; /* Spacing */
+        --rmsc-radius: 4px; /* Radius */
+        --rmsc-h: 38px; /* Height */
+}
```

Comment:
maybe you have this correct but couldn't you remove the comments and be more specific like `--rmsc-spacing` for example?

Reply:
thanks for the comment! I had to go into node_modules and change it in the downloaded component's css, but it works without the comments now! :)